worker/include/RTC/TransportCongestionControlServer.hpp

```diff
@@ -89,6 +92,9 @@ namespace RTC
 		uint8_t unlimitedRembCounter{ 0u };
 		std::deque<double> packetLossHistory;
 		double packetLoss{ 0 };
+		bool receivedTransportWideSeqNumber{ false };
```

Comment:
If this is a boolean it should be `transportWideSeqNumberReveived`. Also I'd appreciate a comment on top explaining what this boolean means.